nodejs/week1/warmup-mealcalculator/src/backend/routes/meal.js

```diff
@@ -0,0 +1,28 @@
+// Respond with the json for a random meal (including it's reviews)
+
+const mealsArray = require("../data/meals.json");
+const reviewsArray = require("../data/reviews.json");
+
+const length = reviewsArray.length;
+
+const getmeal = function (request, response) {
+
+        const randomnum = Math.floor(Math.random() * length) ;
```

Comment:
I'm not sure I understand the logic here. You use `length`, which is the length of `reviewsArray` to get a random index, which you use to access an element from `mealreviewsArray`. Did you want to maybe use the length of `mealsArray` instead? 😉